test/predictions/repo_test.exs

```diff
@@ -58,6 +58,102 @@ defmodule Predictions.RepoTest do
       end
     end
 
+    test "filtes out predictions with no departure" do
+      five_minutes_in_future = DateTime.add(Timex.now(), 5, :minute)
+
+      five_minutes_in_future_string =
+        Timex.format!(five_minutes_in_future, "{ISO:Extended:Z}")
+
+      expect(MBTA.Api.Mock, :get_json, fn _, _ ->
+        %JsonApi{
+          data: [
+            %JsonApi.Item{
+              attributes: %{
+                "departure_time" => nil,
+                "arrival_time" => five_minutes_in_future_string,
+                "direction_id" => 1
+              },
+              relationships: %{
+                "route" => [
+                  %{
+                    id: "Red"
+                  }
+                ],
+                "trip" => [],
+                "vehicle" => [],
+                "stop" => [
+                  %{id: "StopID"}
+                ]
+              }
+            },
+            %JsonApi.Item{
+              attributes: %{
+                "departure_time" => five_minutes_in_future_string,
+                "arrival_time" => five_minutes_in_future_string,
+                "direction_id" => 1
+              },
+              relationships: %{
+                "route" => [
+                  %{
+                    id: "Red"
+                  }
+                ],
+                "trip" => [],
+                "vehicle" => [],
+                "stop" => [
+                  %{id: "StopID"}
+                ]
+              }
+            }
+          ]
+        }
```

Comment:
> These two structs are exactly the same except for that one attribute, so there should be a shorter and clearer way to communicate that.

Because these two structs are identical except for one attribute, you could define it once and then override that attribute for the second. It will make the test shorter. That also helps to communicate the difference between the two.